kedfjjfkjdfgkj

joao1987teste opened the pull request.

snyk-bot committed:
fix: Dockerfile to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-DEBIAN9-APT-407402
- https://snyk.io/vuln/SNYK-DEBIAN9-DPKG-2847943
- https://snyk.io/vuln/SNYK-DEBIAN9-GLIBC-356506
- https://snyk.io/vuln/SNYK-DEBIAN9-SENSIBLEUTILS-323135
- https://snyk.io/vuln/SNYK-DEBIAN9-UTILLINUX-285844

joao1987teste committed:
Merge pull request #1 from joao1987teste/snyk-fix-f2809ac1ce103188c67a93ddbcbc90a2

[Snyk] Security upgrade debian from 9.2 to 12

joao1987teste committed:
Update package.json

Yoavast commented:
![Logo](https://user-images.githubusercontent.com/42570337/152954017-2101d077-2be7-48a8-8d34-618fe5a7c779.png)
**Checkmarx One – Scan Summary & Details** – [24eeb014\-3b4d\-45ec\-b990\-bb060e735c59](https://ast-master-components.dev.cxast.net/projects/7478e6ec-6515-4052-9ea4-55937b5a0049/scans?id=24eeb014-3b4d-45ec-b990-bb060e735c59&branch=main)
### No New Or Fixed Issues Found

snyk-bot committed:
fix: pom.xml to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-JAVA-JUNIT-1017047
- https://snyk.io/vuln/SNYK-JAVA-ORGSPRINGFRAMEWORK-2329097
- https://snyk.io/vuln/SNYK-JAVA-ORGSPRINGFRAMEWORK-2330878
- https://snyk.io/vuln/SNYK-JAVA-ORGSPRINGFRAMEWORK-2434828
- https://snyk.io/vuln/SNYK-JAVA-ORGSPRINGFRAMEWORK-2436751
- https://snyk.io/vuln/SNYK-JAVA-ORGSPRINGFRAMEWORK-2689634
- https://snyk.io/vuln/SNYK-JAVA-ORGSPRINGFRAMEWORK-2823313
- https://snyk.io/vuln/SNYK-JAVA-ORGSPRINGFRAMEWORK-31325
- https://snyk.io/vuln/SNYK-JAVA-ORGSPRINGFRAMEWORK-3369749
- https://snyk.io/vuln/SNYK-JAVA-ORGSPRINGFRAMEWORK-5422217
- https://snyk.io/vuln/SNYK-JAVA-ORGSPRINGFRAMEWORKSECURITY-1078232
- https://snyk.io/vuln/SNYK-JAVA-ORGSPRINGFRAMEWORKSECURITY-31336

joao1987teste committed:
Merge pull request #2 from joao1987teste/snyk-fix-25fa81405a8e629dc59bc027d24d81a0

[Snyk] Fix for 12 vulnerabilities
